### DISCUSSION TITLE
Verhalten Heizstab

### DISCUSSION THREAD

**Discussion by cyberthom42** [cyberthom42/vaillant-arotherm-plus#17] [General]
Ich habe verschiedene Tests mit Heizstab gemacht. Meine Beobachtungen bisher:
Einschalten des Heizstabes durch hochsetzen des Bivalenzpunktes im Heizmodus nicht möglich.
Einachalten des Heizstabes bei WW durch hochsetzen des Bivalenzpunktes möglich, allerdings bleibt der Heizstab weiter an, wenn man den Vivalenzpunkt herunter setzt. Erst beim nächsten WW-Zyklus bleibt der Heizstab aus.
Durch Alternativpunkt kann man die WP abschalten und nur über den Heizstab gehen. 

Ich hatte erst gedacht, dass eine Art Intelligenz gibt, die berechnet welche Heizstableistung notwendig ist. Der Heizstab fährt aber immer auf die maximal eingestellte Leistung hoch. 
Evtl. gibt es ja so eine Regelung im Heizbetrieb.

**Comment by RotorMotor42** [comment]
Das hört sich ja leider nicht so ideal gelöst an.
Wie verhält sich der Heizstab denn, wenn du während dem WW Zyklus den max Wert änderst?

**Comment by RotorMotor42** [comment]
Ich habe mich nochmal mit dem Thema befasst und ein Script geschrieben, dass den Heizstab anhand der PV Leistung regelt.
Leider kann ich während der WW Bereitung nicht zwischen nur WP, Bivalent oder alternativ umschalten.
Besonders interessant wäre für mich Umschalten von bivalent auf nur Heizstab.
Wenn ich aber den Alternativpunkt hochsetze läuft die WP einfach weiter.

@cyberthom42 funktioniert das bei dir?

**Reply by cyberthom42** [reply]
Bei WW: wenn der Heizstab einmal an ist, läßt er sich nicht mehr deaktivieren, man kann aber die Leistung auf 0 setzen. 
Nur Heizstab habe ich noch nicht hinbekommen. 
Im Heizbetrieb ist das Hinzuschalten des Heizstabes auch schwer. Nach meinen bisherigen Tests muss die WP schon bei hoher Last laufen, erst dann schaltet sich der Heizstab dazu, auch wenn der Bivalenzpunkt auf 15 Grad steht.

**Reply by maggo1404** [reply]
Ich vermute wenn der integrale, weiter wegläuft trotz das die Wärmepumpe auf 100% läuft. Wird erst der heizstab dazugeschaltet.

**Comment by cyberthom42** [comment]
Ich bin momentan etwas eingeschränkt, da ich mir die Schulter angebrochen habe :-(
Daher dauert die Antwort etwas länger. 
Hier meine Beobachtung:
Nur Heizstab: Umschaltung über Alternativpunkt, geht auch während schon gestarteter WW Auflade Phase. Ich meine mich zu erinnern, dass man das sogar während des gleichen Ladevorganges wieder zurück schalten kann. Zu beachten sind hier die Vorlauf und nachlaufzeiten der WP (Siehe Statuscode).

Heizstab + WP: lässt sich während der Aufladung mit setzen des Bivalenzpunktes aktivieren, aber NICHT mehr deaktivieren. Um den Heizstab während dieses Ladevorganges wieder zu aktivieren muss man die Leistung auf 0 setzen (das geht zum Glück). Die Leistung lässt sich während der ganzen Zeit immer wieder anpassen, die Reaktion ist sogar relativ schnell.

**Reply by RotorMotor42** [reply]
Danke für die Antwort und gute Besserung!

Ja, das setzen der Heizstableistung im Bivalenzmodus nutze ich schon aktiv um PV strom reinzuschieben.
Alternativpunkt reagiert bei mir leider gar nicht.

**Reply by cyberthom42** [reply]
Ich werde das bei Sonne noch einmal strukturierter testen

**Reply by swi88** [reply]
Hallo, 
ich würde auch gerne über des setzen des Bivalenzpunktes, oder noch besser des Alternativpunktes die Nutzung des Heizstabs bei meiner Arotherm Plus 55/6 erzwingen. 
Leider finde ich nirgends Infos über die Adresse in der csv. Kann mir da jemand weiterhelfen?

**Reply by chrizzzp** [reply]
Hallo,

Hier die Adressen, wie sie für den Regler VRC720/2 bei mir funktionieren (15.ctlv2.csv):

```
*r,,,,,,B524,02000000,,,IGN:4,,,
*w,,,,,,B524,02010000,,,,,,
r;w,,HwcBivalencePoint,,,,,0100,,,tempv,,,Bivalence point hot water heating
r;w,,HcAlternativePoint,,,,,2200,,,tempv,,,exclusively alternate heating below alternative point (-21=off)
r;w,,HcBivalencePoint,,,,,2300,,,tempv,,,Bivalence point heating
```

Über den Alternativpunkt kann man recht einfach den Heizstab einschalten, sofern eine aktuelle Wärmeanforderung anliegt.
Den Alternativpunkt für Warmwasser (Hwc) gibt es leider erst in der neusten Version des Reglers (VRC720/3).

**Reply by swi88** [reply]
Perfekt, vielen Dank! Die funktionieren.
Ich besitze auch den VRC720/2. Bei mir wird jedoch nur die 15.720.csv geladen. Da muss ich bei Gelegenheit mal nachsehen.

**Comment by cyberthom42** [comment]
Hier noch mal kurz eine Zusammenfassung:
Alternativpunkt scheint nur bei Heizung zu funktionieren, jedoch nicht bei WW.
Mit dem Bivalenzpunkt wiederum kann man nur bei WW den Heizstab hinzuschalten. Im Heizbetrieb schaltet sich bei der Heizstab erst zu (bei erreichten Bivalenzpunkt), wenn es die WP nicht mehr schafft (gewisse Zeit auf 100%) .

**Comment by cyberthom42** [comment]
Ich hatte jetzt bei den kalten Temperaturen (0 Grad) eine interessante Bebnachtung. Ich nutze öfter den Heizstab bei Warmwasser, wenn ich kurzfristig schnell warmes Wasser benötige (Badewann). Dabei blieb bisher immer der Kompressor weiter an und der Heizstab schaltete sich dazu. Jetzt hatte ich beobachtet (Bivalenzpunkt erhöht hatte um den Heizstab zu aktivieren), dass der Kompressor abgeschaltet hat:
<img width="1920" height="1080" alt="grafik" src="https://github.com/user-attachments/assets/d88d76c0-9d7c-4385-9463-fc2a75739432" />

**Reply by chrizzzp** [reply]
Sind Deine WW-Takte sonst kürzer? Ist vielleicht die maximale WW-Ladezeit erreicht worden (warum auch immer der Heizstab dann davon nicht betroffen wäre):

```
r;w,,HwcMaxChargeTime,maximale hot water charging time,,,,1700,,,minutes2,,,maximum cylinder charging time (20-120 15=off)
```
